books/spiders/books.py

```diff
@@ -0,0 +1,53 @@
+import scrapy
+from scrapy.http import Response
+from selenium import webdriver
+
+
+class BooksSpider(scrapy.Spider):
+    name = "books"
+    allowed_domains = ["toscrape.com"]
+    start_urls = ["https://books.toscrape.com/"]
+
+    text_to_number = {
+        "Zero": 0,
+        "One": 1,
+        "Two": 2,
+        "Three": 3,
+        "Four": 4,
+        "Five": 5,
+    }
+
+    def __init__(self, **kwargs) -> None:
+        super().__init__(**kwargs)
+        self.driver = webdriver.Chrome()
+
+    def close(self, reason: any) -> None:
+        self.driver.close()
+
+    def parse(self, response: Response, **kwargs) -> None:
```

Comment:
Good job, but don't forget to use the correct type of annotations, It wouldn't be None